engine/sdk/src/test/wscript

```diff
@@ -111,7 +114,7 @@ def build(bld):
             content = template.replace('{custom_include}', include)
             f.write(content)
 
-        bld.stlib(features = 'cxx sdk_include_filter',
+        bld.stlib(features = 'cxx sdk_include_filter test',
```

Comment:
By adding "test", their output won't get installed into the DYNAMO_HOME folders.